cfpb/consumerfinance.gov
# Know before you owe: Update lightbox

## Changes

- Update lightbox from 2.11.0 to 2.11.3.


## How to test this PR

1. Pull branch, run ./frontend.sh and visit http://localhost:8000/know-before-you-owe/compare/ and see that clicking the document images still shows a lightbox image.


## Screenshots

<img width="998" alt="Screen Shot 2020-12-16 at 4 56 37 PM" src="https://user-images.githubusercontent.com/704760/102411286-b2713880-3fbf-11eb-946e-bfdb69ae9f6e.png">
